app/scripts/controllers/logs.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+/* global moment: false */
+
+angular.module('openhimConsoleApp')
+  .controller('LogsCtrl', function ($scope, $location, Api) {
+    function formatLog(log) {
+      return new moment(log.timestamp).format('YYYY-MM-D HH:mm:ss.SSS') + ' - ' + log.level + ': [' + log.label + '] ' + log.message + '\n';
+    }
+
+    var lastFetch;
+    var locParams = $location.search();
+    $scope.params = angular.equals({}, locParams) ? { level: 'info' } : locParams;
+    $scope.logs = '';
+
+    if ($scope.params.from || $scope.params.until) {
+      $scope.autoupdate = false;
+      $scope.tailLogs = false;
+    } else {
+      $scope.autoupdate = true;
```

Comment:
I could but I I didn't want to give the user too much choice. Also, autoupdate only functions when there isn't a date range set so would be disabled a lot of the time. Also with tailing off its easy to keep you place as logs update automatically as your view won't be reset. I don't mind changing it, but I'm happy with how it works atm. So, let me know how strongly you feel about this.